src/main/java/games/strategy/engine/ClientContext.java

```diff
@@ -50,6 +54,8 @@
   private final BattleCalcSettings battleCalcSettings;
   private final BattleOptionsSettings battleOptionsSettings;
 
+  private final DownloadCoordinator downloadCoordinator = new DownloadCoordinator();
```

Comment:
I inlined the initialisation of all fields to be consistent, as a constructor is not needed here